### ISSUE TITLE
semver-checks?

### ISSUE THREAD

**Issue by peter-kehl** [epage/_rust#9]
I know you, cargo team & Predrag are working toward incorporating `cargo-semver-checks` into cargo (https://github.com/obi1kenobi/cargo-semver-checks/issues/61). But would it make sense to make your template use `cargo-semver-checks` or https://github.com/obi1kenobi/cargo-semver-checks-action?

Also, FYI
- https://jstrong.dev/posts/2023/publishing-crates-a-justfile-workflow
- https://news.ycombinator.com/item?id=29010327 -> https://fasterthanli.me/articles/my-ideal-rust-workflow.

**Comment by epage** [comment]
Last I looked at cargo-semver-checks, the action expects you to bump the version during development, rather than at release, which is counter to my workflow (and a lot of other workflows seeing as cargo-release was changed to be like this).

I use cargo-release for my releases and it manages a lot of complex cases a simple justfile wouldn't handle.  I am looking at release-plz for some specific cases and it has a lot of limitations.
